### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by hzxcaryn**
### Changes
Added the landing page, react-router-dom, media query constants and removed some useless cra generated files.
Apparently the landing page styles are more complicated that I expected HAHA but I got it to look like the mockup in the figma already :) 

#### Web Screenshot
screencapture-localhost-3000-2019-10-15-05_25_19

#### Mobile Screenshot
screencapture-localhost-3000-2019-10-15-05_26_17

### How to test
`yarn start` in `web` and you should see the landing page!

### CODE REVIEW

File: Web/src/Routes.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { lazy, Suspense } from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+const LandingPage = lazy(() => import('./components/landing'));
```
Comment (indocomsoft) [inline Web/src/Routes.tsx]: <3 lazy loading ftw

File: Web/src/utils/media.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const smMin = '576px';
```
Comment (indocomsoft) [inline Web/src/utils/media.ts]: Magic numbers hahaha

Maybe can say taken from bootstrap

File: Web/src/Routes.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+const Routes: React.FC = () => (
+  <Suspense fallback={<div>Loading...</div>}>
```
Comment (indocomsoft) [inline Web/src/Routes.tsx]: Not sure if you wanna stylize the fallback

File: Web/src/components/landing/IntroductionContent.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+`;
+
+const IntroductionContent: React.FC = () => (
+  <ContentDiv>
+    <p>
+      Can’t decide whose turn it is to do the chores? Or pay for the next
+      outing?
+    </p>
+    <p>
+      Decide now with FAST and SIMPLE mini game with <BrandText />.
```
Comment (indocomsoft) [inline Web/src/components/landing/IntroductionContent.tsx]:
```suggestion
      Decide now with FAST and SIMPLE mini games with <BrandText />.
```

File: Web/src/components/landing/index.tsx [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+  display: none;
+  transform: translateY(15px);
+
+  @media (max-width: ${mdMin}) {
+    display: block;
+  }
+`;
+
+const LandingPage: React.FC = () => (
+  <LandingContatiner>
```
Comment (indocomsoft) [inline Web/src/components/landing/index.tsx]: Contatiner lol

File: Web/src/utils/media.ts (hunk 2 above)
Comment (hzxcaryn) [reply Web/src/utils/media.ts]: Sure

File: Web/src/Routes.tsx (hunk 3 above)
Comment (hzxcaryn) [reply Web/src/Routes.tsx]: Haha I'm intending to to that in the next PR :)

File: Web/src/components/landing/index.tsx (hunk 5 above)
Comment (hzxcaryn) [reply Web/src/components/landing/index.tsx]: HAHAHHA fug I'm sorry brain fart

File: Web/src/Routes.tsx (hunk 3 above)
Comment (indocomsoft) [reply Web/src/Routes.tsx]: 👍

**Pull request merged**